packages/solver-sat/src/solver-sat.ts

```diff
@@ -507,16 +507,27 @@ export const ELEMENT_HANDLERS = {
             const cellCoords = arrayObj2array(renbanCells || {}).map(idx => cellIdx2cellCoord(idx, context.grid));
             if (0 >= cellCoords.length) continue;
 
-            // 1: Encode no repeats.
-            numLits = encodeNoRepeats(numLits, cellCoords, context);
-
-            // 2: CREATE LITERALS to mark if V is in the region.
+			// 1: Ensure that no cell is included multiple times.
+			const uniqueCoords = cellCoords.filter(
+				(n,i, arr) => {
+					return arr.findIndex(
+						t => {
+							if(n === t)
+								return true;
+							return n[0] === t[0] && n[1] === t[1];
+						}) === i;
+				})
```

Comment:
`const uniqueCoords = Array.from(new Set(cellCoords));` is simpler (untested)

Reply:
I tested this, and it did not work.

Presumably it's because Arrays with the same elements do not test equal.

Reply:
Mhm yes forgot about array equality